docker/pypi/wmagent/wmagent-docker-run.sh

```diff
@@ -112,12 +112,17 @@ $tnsMount \
 --mount type=bind,source=/etc/sudoers.d,target=/etc/sudoers.d,readonly \
 "
 
+registry=local
+repository=wmagent
 $PULL && {
+    registry=registry.cern.ch
+    project=cmsweb
+    repository=wmagent
     echo "Pulling Docker image: registry.cern.ch/cmsweb/wmagent:$WMA_TAG"
     docker login registry.cern.ch
-    docker pull registry.cern.ch/cmsweb/wmagent:$WMA_TAG
-    docker tag registry.cern.ch/cmsweb/wmagent:$WMA_TAG local/wmagent:$WMA_TAG
-    docker tag registry.cern.ch/cmsweb/wmagent:$WMA_TAG local/wmagent:latest
+    docker pull $registry/$project/$repository:$WMA_TAG
```

Comment:
I thought I have parametrized the local vs. remote tags properly for all of types of container we have, but obviously I've forgotten the wmagent one.